### PR TITLE
Debugging html-proofer - build "passing" now

### PR DESCRIPTION
**Pull request by amragaey**
I fixed all issues in html-proofer and make a passing build on **Travis**, except the Debian package is commented as I haven't found a solution so far.

### CODE REVIEW

File: Support/BoneScript/knight-rider._html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (jadonk) [inline Support/BoneScript/knight-rider._html]: What is this?

Comment (amragaey) [inline Support/BoneScript/knight-rider._html]: This is unused page causing errors in htmlproofer, just renamed it to ._html

File: _includes/side_menu.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <h2>{{ include.title }}</h2>
 <div id='side-menu' class="t3_sidebar">
     <ul class="left-menu"><strong>Navigation</strong>
-        <li><a href="{{ site.baseurl }}/static/side-menu.html">Menu contents</a></li>
```
Comment (jadonk) [inline _includes/side_menu.html]: Why remove this link? Formatting?

File: Support/bone101/index.html [hunk 3]
```diff
@@ -170,7 +170,7 @@ <h2 id="node-red"><a target="_blank" class="node-red-url" href="http://192.168.7
  <h2 id="cloud9"><a target="_blank" class="cloud9-url" href="http://192.168.7.2:3000">Cloud9 IDE</a></h2>
  <p>To begin editing programs that live on your board, you can use the Cloud9 IDE.</p>
  <p>Click on the "Cloud9 IDE" link above to start the editor.</p>
- <a target="_blank" class="cloud9-url">
+ <a target="_blank" class="cloud9-url" href="http://192.168.7.2:3000">
```
Comment (jadonk) [inline Support/bone101/index.html]: We'll need to look at the replacement script for this URL such that this is the IP address of the board where we are connected.

File: _includes/side_menu.html (hunk 2 above)
Comment (amragaey) [inline _includes/side_menu.html]: this link isn't valid

File: Support/bone101/index.html (hunk 3 above)
Comment (amragaey) [inline Support/bone101/index.html]: I will consider it

File: _includes/side_menu.html (hunk 2 above)
Comment (jadonk) [inline _includes/side_menu.html]: It should be valid. This should be the location the menu is pulled from. The big reason I have it there is for web crawlers. We want the content to be discovered.

File: Support/bone101/index.html (hunk 3 above)
Comment (amragaey) [inline Support/bone101/index.html:173]: solved it

File: _includes/side_menu.html (hunk 2 above)
Comment (amragaey) [inline _includes/side_menu.html:4]: recovered, but produces errors in html-proofer

**Pull request merged**